src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        for (int i = 0; i < 3; i++) {
```

Comment:
3 is a magic number, create constant

Reply:
but if I make a constant like (final int LOTTERY_BALL_COUNT = 3;) maven does not pass such code because it considers it incorrect